src/main/java/lotto/model/Lotto.java

```diff
@@ -0,0 +1,43 @@
+package lotto.model;
+
+import java.util.HashSet;
+import java.util.List;
+
+public class Lotto {
+    private final List<Integer> numbers;
+
+    public Lotto(List<Integer> numbers) {
+        validate(numbers);
+        this.numbers = numbers;
+    }
+
+    private void validate(List<Integer> numbers) {
+        validateSize(numbers);
+        validateRange(numbers);
+        validateDuplicate(numbers);
+    }
+
+    private void validateSize(List<Integer> numbers) {
```

Comment:
```suggestion
    private void validateSize(List<Integer> numbers) {
        if (numbers.size() != 6) {
            throw new IllegalArgumentException("[ERROR] 로또 번호는 6개여야 합니다.");
        }
    }

    private void validateRange(List<Integer> numbers) {
        for (Integer number : numbers) {
            if (number < 1 || number > 45) {
                throw new IllegalArgumentException("[ERROR] 로또 번호는 1부터 45 사이의 숫자여야 합니다.");
            }
        }
    }

    private void validateDuplicate(List<Integer> numbers) {
        if (new HashSet<>(numbers).size() != numbers.size()) {
            throw new IllegalArgumentException("[ERROR] 로또 번호는 중복되지 않아야 합니다.");
        }
    }
```

컴팩트하게 잘 짜여진 코드네요!
저같은 경우는 validate 관련 파트를 validator 패키지를 만들고 해당 부분에 정리해봤어요.
따로 validator를 만들었으니 거기에 모아두는게 더 유지 보수에 유리할 수도 있다고 생각합니다!
혹시 혜정님은 어떻게 생각하시나요?